docs/visualization/visualize_unit_metrics.ipynb

```diff
@@ -107,14 +107,14 @@
      "name": "stderr",
```

Comment:
<p>he "noise" units are ones that are suspected not to be actual neurons or "to be poor quality recordings of one or more neurons"</p><p> </p><p>"slightly obfuscated way" suggest some nefarious intent. Perhaps replace with "and is indirectly stored within the NWB..."</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AllenInstitute/openscope_databook/pull/139/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>